src/handlers/miner_hbbft_handler.erl

```diff
@@ -214,15 +214,22 @@ handle_command({txn, Txn}, State=#state{hbbft=HBBFT}) ->
     Buf = hbbft:buf(HBBFT),
     case lists:member(blockchain_txn:serialize(Txn), Buf) of
         true ->
+            %% TODO return the existant position in Buf
             {reply, ok, ignore};
         false ->
-            case hbbft:input(State#state.hbbft, blockchain_txn:serialize(Txn)) of
-                {NewHBBFT, ok} ->
-                    {reply, ok, [], State#state{hbbft=NewHBBFT}};
+            Fee = blockchain_txn:fee(Txn),
```

Comment:
I think we should give state channel closes a `virtual` fee here so they sort before poc txns, so router can get them closed more easily